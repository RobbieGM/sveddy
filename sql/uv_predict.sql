DROP SCHEMA public CASCADE;
CREATE SCHEMA public;
GRANT ALL ON SCHEMA public TO current_user;
GRANT ALL ON SCHEMA public TO public;
CREATE EXTENSION sveddy;


SELECT predict_uv('{0.1, 0.2, 0.3, 0.4, 0.5, 0.6}', '{0.4, 0.5, 0.6, 0.7, 0.8, 0.9}');
