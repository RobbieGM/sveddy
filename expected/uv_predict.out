DROP SCHEMA public CASCADE;
NOTICE:  drop cascades to 4 other objects
DETAIL:  drop cascades to extension sveddy
drop cascades to table ratings
drop cascades to table ratings_sveddy_model_u
drop cascades to table ratings_sveddy_model_v
CREATE SCHEMA public;
GRANT ALL ON SCHEMA public TO current_user;
GRANT ALL ON SCHEMA public TO public;
CREATE EXTENSION sveddy;
SELECT predict_uv('{0.1, 0.2, 0.3, 0.4, 0.5, 0.6}', '{0.4, 0.5, 0.6, 0.7, 0.8, 0.9}');
 predict_uv 
------------
       1.54
(1 row)

